Feature/5

StanislavDargel opened the pull request.

StanislavDargel committed:
#AssertionError

StanislavDargel committed:
#feature5
1. Fixed some minor mistakes.
2. In <validation> package exception messages been changed.
3. Project covered with 69% tests.

StanislavDargel committed:
#feature5
1. Fixed some minor mistakes.
2. In <validation> package exception messages been changed.
3. Project covered with 69% tests.

StanislavDargel committed:
#feature5
1. Fixed BeanMapperTest.java, ProductRepositoryTest.java, ValidationServiceTest.java;

javagururt commented on src/test/java/com/javaguru/shoppinglist/mapper/BeanMapperTest.java:
```diff
@@ -1,67 +1,67 @@
 
+@RunWith(MockitoJUnitRunner.class)
 public class BeanMapperTest {
+    private final ProductDTO DTO = new ProductDTO();
```
это не константы

javagururt replied:
лучше вынести создание и заполнение в отдельный метод

StanislavDargel committed:
#feature6
1. New dependencies added in project. (Spring framework);
2. Were created new package <configuration> and new class ApplicationConfiguration.java;
3. New CloseProgram.java was created, "Exit" method moved from ConsoleUI.java into CloseProgram.java;
4. CreateProduct.java action method was splitted into new classes:
-> ProductCategoryInput.java;
-> ProductDescriptionInput.java;
-> ProductDiscountInput.java;
-> ProductNameInput.java;
-> ProductPriceInput.java;
new Interface ProductParameters.java was added;
5. Minor fixes: Test final variables changed into ordinary variables;
6. Method toString were added into ProductCategory.java, CreateProduct.java, EditParameters.java, FindProductById.java and RemoveProduct.java;
7. Dependencies injections changed by Spring framework into ShoppingListApplication.java;
8. void printProductInfo(...) method changed into String printProductInfo(...) method into ValidationService.java;
